packages/cephalon/src/actions/record-speaker.scope.ts

```
@@ -2,7 +2,8 @@ import type * as discord from "discord.js";
 
 import type { Bot } from "../bot.js";
 import { makeLogger, type Logger } from "../factories/logger.js";
-import { makePolicy, type PolicyChecker } from "../factories/policy.js";
+import { makePolicy, type PolicyChecker } from "@promethean/security/policy.js";
```

ESLint

[error] `@promethean/security/policy.js` import should occur before type import of `../bot.js`

(import/order)